eng/build.ps1

```diff
@@ -364,6 +364,10 @@ function TestUsingRunTests() {
     }
   }
 
+  if ($ci) {
+    $env:ROSLYN_CI = "true"
```

Comment:
Should remove this environment variable as we do the other env vars below. 

If we're doing this for only tests I'd consider naming it `%ROSLYN_TEST_CI%` but don't feel too strongly about that.

Reply:
➡️ Fixed!